backstop_data/engine_scripts/puppet/page-objects/crm-page.js

```diff
@@ -98,6 +99,20 @@ module.exports = class CrmPage {
     await Promise.all([
       this.engine.click(selector),
       this.engine.waitForNavigation()
-    ]);      
+    ]);
+  }
+
+  /**
+   * Checks if there are no price fields present on the page or not
+   *
+   */
+  async checkIfPriceFieldsAreEmpty () {
+    // checks for the messages div - appears if the list is empty.
+    const priceFieldsEmpty = await this.engine.$('#crm-main-content-wrapper .messages');
```

Comment:
are you sure this selector is reliable enough? there are no other elements on the page that match the same selector?

Reply:
Yes! I double checked it.